bundles/SimpleBackendSearchBundle/src/Migrations/Version20230914085710.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\SimpleBackendSearchBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230914085710 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Change some columns of search_backend_table to non nullable';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE `search_backend_data` MODIFY `fullpath` varchar(765) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL');
```

Comment:
These properties are non nullable in db but i can still initialize `Backend\Data` object with these properties as null since it is not set in `__construct`

Reply:
@bramstroker  🏓😊 Thanks!

Reply:
@bramstroker Did you maybe have a chance to look at the requested change? Otherwise we would close the pull request soon.